src/app/app.component.html

```diff
@@ -5,31 +5,17 @@
         </ecl-site-header-action>
 
         <ecl-menu>
-            <ecl-menu-item label="Home"></ecl-menu-item>
-            <ecl-menu-item label="Item 1">
-                <ecl-menu-mega>
-                    <ecl-menu-mega-item label="Item 1.1"></ecl-menu-mega-item>
-                    <ecl-menu-mega-item label="Item 1.2"></ecl-menu-mega-item>
-                    <ecl-menu-mega-item label="Item 1.3" [isCurrent]="true"></ecl-menu-mega-item>
-                </ecl-menu-mega>
-
-            </ecl-menu-item>
-
-            <ecl-menu-item label="Item 2">
-                <ecl-menu-mega>
-                    <ecl-menu-mega-item label="Item 2.1"></ecl-menu-mega-item>
-                    <ecl-menu-mega-item label="Item 2.2"></ecl-menu-mega-item>
-                    <ecl-menu-mega-item label="Item 2.3"></ecl-menu-mega-item>
-                </ecl-menu-mega>
-            </ecl-menu-item>
-
-            <ecl-menu-item label="Item 3">
-                <ecl-menu-mega>
-                    <ecl-menu-mega-item label="Item 3.1"></ecl-menu-mega-item>
-                    <ecl-menu-mega-item label="Item 3.2"></ecl-menu-mega-item>
-                    <ecl-menu-mega-item label="Item 3.3"></ecl-menu-mega-item>
-                </ecl-menu-mega>
-            </ecl-menu-item>
+            <ng-container *ngFor="let menuItem of menuItems">
```

Comment:
ok, it's already something to have support for children items, but a menu item can have something more than just the label and the path, in ECL and i guess in eUI as well we do have "external" as a way to mark the item as external (the external icon appears) and is_current to mark the current item, can you check how this is supported in eUI?